WHERE_TO_GO_NEXT.md

```diff
@@ -0,0 +1,6 @@
+For LABS_38 Backend Team From LABS_37:
+
+There was a lot of confusion at the beginning of our labs month as to what was going on with the actual tables of this database.
+
+Firstly, there is one database and it is being created by code inside the data science repo for this project. The code inside of this backend repo interacts with their 'force_ranks' table and their 'conversations' table at the 
+
```

Comment:
Love the advice you're giving the next cohort. Seems like this will make for a great hand off.